README.adoc

```diff
@@ -1,10 +1,8 @@
 = Gravitee.io Common
 
-ifdef::env-github[]
-image:https://ci.gravitee.io/buildStatus/icon?job=gravitee-io/gravitee-common/master["Build status", link="https://ci.gravitee.io/job/gravitee-io/job/gravitee-common/"]
-image:https://f.hubspotusercontent40.net/hubfs/7600448/gravitee-github-button.jpg["Join the community forum", link="https://community.gravitee.io?utm_source=readme", height=20]
-endif::[]
-
+image:https://img.shields.io/badge/License-Apache%202.0-blue.svg["License", link="https://github.com/gravitee-io/gravitee-common/blob/master/LICENSE.txt"]
+image:https://img.shields.io/badge/semantic--release-conventional%20commits-e10079?logo=semantic-release["Releases", link="https://github.com/gravitee-io/gravitee-common/releases"]
+image:https://circleci.com/gh/gravitee-io/gravitee-common.svg?style=svg["CircleCI", link="https://circleci.com/gh/gravitee-io/gravitee-common"]
```

Comment:
image:https://f.hubspotusercontent40.net/hubfs/7600448/gravitee-github-button.jpg["Join the community forum", link="https://community.gravitee.io?utm_source=readme", height=20]

Reply:
fixed